pkgs/development/python-modules/httpie-ntlm/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, httpie
+, requests_ntlm
+, isPy39
+}:
+
+buildPythonPackage rec {
+  pname = "httpie-ntlm";
+  version = "1.0.2";
+  format = "setuptools";
+  disabled = ! isPy39; # Need same python version as httpie application
```

Comment:
Please use pythonOlder/pythonAtLeast

Reply:
> Please use pythonOlder/pythonAtLeast

Thanks @SuperSandro2000 for improvement.